sos/report/plugins/kubernetes.py

```diff
@@ -238,6 +270,26 @@ def setup(self):
         if self.is_installed('microk8s'):
             self.kube_cmd = 'microk8s kubectl'
 
+        self.config_files.extend([
+            '/root/cdk/kubelet/config.yaml',
+            '/root/cdk/audit/audit-policy.yaml'
+        ])
+        if self.get_option('all'):
+            self.global_resources.extend([
+                'roles',
+                'rolebindings'
+            ])
+            self.resources.extend([
+                'endpoints',
+                'statefulsets',
+                'configmaps',
+                'serviceaccounts',
+                'secrets',
+                'jobs',
+                'cronjobs',
+                'clusterroles',
+                'clusterrolebindings'
+            ])
```

Comment:
Are these actually specific to the Ubuntu packaging for kubernetes? If not, they should be in the base class. I have no real opinion on gating them behind the `all` option, but am curious if these aren't usually desired for troubleshooting?

Reply:
I've moved them to the base class.
They are less common/important and can save time to dump less data.
We can move them around based on feedback.